docs/README.md

```diff
@@ -9,489 +10,579 @@
+*Defined in [constants.ts:45](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/constants.ts#L45)*
 
 Keccak-256 hash of the RLP of null
 
 ___
-<a id="max_integer"></a>
 
-### `<Const>` MAX_INTEGER
+### `Const` MAX_INTEGER
 
-**● MAX_INTEGER**: *`BN`* =  new BN(
+• **MAX_INTEGER**: *[BN](README.md#bn)* = new BN(
   'ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff',
   16,
 )
 
-*Defined in [constants.ts:6](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/constants.ts#L6)*
+*Defined in [constants.ts:7](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/constants.ts#L7)*
 
 The max integer that this VM can handle
 
 ___
-<a id="two_pow256"></a>
 
-### `<Const>` TWO_POW256
+### `Const` TWO_POW256
 
-**● TWO_POW256**: *`BN`* =  new BN(
+• **TWO_POW256**: *[BN](README.md#bn)* = new BN(
   '10000000000000000000000000000000000000000000000000000000000000000',
   16,
 )
 
-*Defined in [constants.ts:14](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/constants.ts#L14)*
+*Defined in [constants.ts:15](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/constants.ts#L15)*
 
 2^256
 
 ___
-<a id="publictoaddress"></a>
 
-### `<Const>` publicToAddress
+### `Const` assert
+
+• **assert**: *any* = require('assert')
+
+*Defined in [account.ts:1](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/account.ts#L1)*
+
+*Defined in [object.ts:1](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/object.ts#L1)*
+
+___
+
+### `Const` createHash
 
-**● publicToAddress**: *[pubToAddress]()* =  pubToAddress
+• **createHash**: *any* = require('create-hash')
+
+*Defined in [hash.ts:2](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/hash.ts#L2)*
+
+___
+
+### `Const` createKeccakHash
+
+• **createKeccakHash**: *any* = require('keccak')
+
+*Defined in [hash.ts:1](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/hash.ts#L1)*
+
+___
+
+### `Const` ethjsUtil
+
+• **ethjsUtil**: *any* = require('ethjs-util')
+
+*Defined in [bytes.ts:1](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/bytes.ts#L1)*
+
+*Defined in [hash.ts:3](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/hash.ts#L3)*
+
+*Defined in [account.ts:2](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/account.ts#L2)*
+
+*Defined in [object.ts:2](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/object.ts#L2)*
+
+*Defined in [index.ts:2](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/index.ts#L2)*
+
+___
+
+### `Const` publicToAddress
+
+• **publicToAddress**: *pubToAddress* = pubToAddress
 
 *Defined in [account.ts:163](https://github.com/ethereumjs/ethereumjs-util/blob/master/src/account.ts#L163)*
 
 ___
-<a id="setlength"></a>
 
-### `<Const>` setLength
+###  rlp
 
-**● setLength**: *[setLengthLeft]()* =  setLengthLeft
+• **rlp**: *"/Users/ryanghods/dev/ethereumjs-util/node_modules/rlp/dist/index"*
```

Comment:
Huh this is funny. It looks like it is trying to automatically document this line `import rlp = require('rlp')` and accidentally inserting my file system path. Didn't seem to happen anywhere else though. I'll see if there is any way to solve it.

Reply:
Hi Ryan, thanks for having a start on this. Yeah, this is what I meant in the [issue description comment](https://github.com/ethereumjs/ethereumjs-util/issues/230#issue-556824987) in the last paragraph mentioning the `require` syntax used.

This would need some deeper analysis.

Some possible ideas:
- Eventually it is possible to replace some of this special `require` syntax includes with normal `import` statements (would be best)
- There might be a `TypeDoc` option I have missed to prevent this from being included in the docs (second best)
- Eventually there is some for of comment syntax to be placed above the respective `require` statements to indicate to `TypeDoc` to not to include the lines (third best, no problem though either)